packages/front-end/components/Settings/UpgradeModal/index.tsx

```diff
@@ -257,6 +285,154 @@ export default function UpgradeModal({ close, source }: Props) {
     }
   };
 
+  const bullets: Partial<Record<CommercialFeature, string>> = {
+    "advanced-permissions": "Manage advanced user permissions",
+    "encrypt-features-endpoint": "Protect your SDK with endpoint encryption",
+    "schedule-feature-flag": "Schedule feature flag rollouts",
+    "override-metrics": "Override your metrics default behavior",
+    "regression-adjustment":
+      "Increase experiment velocity with regression adjustment (CUPED)",
+    "sequential-testing": "Run sequential tests for better insights",
+    "visual-editor": "Edit feature flags with our visual editor",
+    archetypes: "Build user archetype for reusable feature flag targeting",
+    simulate: "Simulate features and experiments a user would be exposed to",
+    "cloud-proxy": "Leverage a cloud proxy for seamless SDK integration",
+    "hash-secure-attributes": "Secure sensitive attributes with hashing",
+    livechat: "Get fast support with in-app chat",
+    "remote-evaluation": "Evaluate features remotely in real time",
+    "sticky-bucketing": "Ensure consistent experiences with sticky bucketing",
+    "code-references":
+      "Quickly see where feature flags are leveraged in your codebase",
+    prerequisites: "Set prerequisites to control feature rollouts",
+    redirects: "Run URL Redirect tests",
+    "multiple-sdk-webhooks": "Hook into Growthbook's SDK Webhooks",
+    "quantile-metrics": "Analyze data using quantile-based metrics",
+    "retention-metrics": "Boost retention with precise tracking",
+    "metric-populations": "Segment users using metric insights",
+    "multi-armed-bandits": "Run adaptive experiments with Bandits",
+    "historical-power": "Leverage historical data for power analysis",
+    "mid-experiment-power": "Assess mid-experiment power for quick adjustments",
```

Comment:
@lukesonnet @ahdriel Morning! I saw on a recent PR we switched the `CommercialFeature` from `mid-experiment-power` to`decision-framework`. I'm working on bullet points for the upgrade modal. 

Do you have any suggestions on the bullet point text for the new `decision-framework` CommercialFeature - I know there are going to be stages to what's included in the DMF so I don't want to oversell it in its current state.

Thanks!

P.S. @lukesonnet I know you're close to our customers, so if any of these other bullet points could be improved, I'm all ears!